hometask6

```diff
@@ -0,0 +1,68 @@
+# Task 1
+#
+# import time
+# def my_decorator(func):
+#     def wrap():
+#         print(time.ctime())
+#         func()
```

Comment:
It is better to use `return` statement here and move this line to the end of the function. This will alow you to decorate functions that return some value